nodejs/nodejs-cloud-run-hello-world/package-lock.json

```diff
@@ -914,7 +914,7 @@
         "extend": "^3.0.2",
         "https-proxy-agent": "^5.0.0",
         "is-stream": "^2.0.0",
-        "node-fetch": "^2.3.0"
+        "node-fetch": "^2.6.1"
```

Comment:
A previous lock file maintenance erroneously down-dated the "node-fetch" version from 2.6.1 to 2.3.0.
I committed the correct version (2.6.1) in this PR.
Keeping this commit in here instead of opening a separate PR for simplicity's sake.